src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/InterproceduralAnalysisConfiguration.cs

```diff
@@ -40,16 +40,16 @@ public struct InterproceduralAnalysisConfiguration : IEquatable<InterproceduralA
         public static InterproceduralAnalysisConfiguration Create(
             AnalyzerOptions analyzerOptions,
             DiagnosticDescriptor rule,
-            ISymbol symbol,
+            ControlFlowGraph cfg,
```

Comment:
This is the core change, everything else in the PR is just boiler plate changes.